app/Eloquent/UpdateBook.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace App\Eloquent;
+
+use Illuminate\Database\Eloquent\Model;
+
+class UpdateBook extends Model
+{
+    protected $table = 'update_books';
+
+    protected $fillable = [
+    	'title',
+    	'description',
+    	'author'
+    	'pulish_date',
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Generic.WhiteSpace.DisallowTabIndent.TabsUsed</br>**Severity**: error</br>**File**: app/Eloquent/UpdateBook.php</br>Spaces must be used to indent lines; tabs are not allowed